### PR TITLE
New package: diffuse-0.7.5

### LINKED ISSUE #39398: Package request: diffuse
### Package name

diffuse

### Package homepage

https://mightycreak.github.io/diffuse/

### Description

Graphical tool for merging and comparing text files

### Does the requested package meet the package requirements?

System

### Is the requested package released?

Yes

### PR DESCRIPTION
**Pull request by drichline**
Closes #39398 

#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures:
  - aarch64-glibc

### CODE REVIEW

File: srcpkgs/diffuse/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+makedepends="python3-cairo-devel python3-gobject-devel"
+short_desc="Graphical tool for merging and comparing text files"
+maintainer="Dakota Richline <drichline@protonmail.com>"
+license="GPL-2.0-only, FSFAP"
+homepage="https://github.com/MightyCreak/diffuse"
+distfiles="https://github.com/MightyCreak/diffuse/archive/refs/tags/v${version}.tar.gz"
+checksum=df34a7b2842956dc397d8e72f4d744dc50d403b515c5611f54bd137f3554eeb3
+
+post_install() {
+	vlicense COPYING
```
Comment (abenson) [inline srcpkgs/diffuse/template:16]: These aren't needed for GPL.

Comment (drichline) [reply srcpkgs/diffuse/template]: Thanks! Just to clarify: no license information needs to be installed for GPL packages?

Comment (abenson) [reply srcpkgs/diffuse/template]: Correct.

**Comment by drichline** [comment]
Sorry, can I get a review on this? It should hopefully be ready to merge.

**Review by paper42 — COMMENTED** [review]
at least `gtk+3` and `python3-gobject` runtime dependencies are missing

**Comment by drichline** [comment]
> at least `gtk+3` and `python3-gobject` runtime dependencies are missing

I'm really sorry, I forgot to get back on this issue. I've also added python3-cairo since it might be a runtime dependency (?)

**Pull request merged**